makefile

```diff
@@ -1,11 +1,12 @@
 # Create the docker images locally. If a BUILD_NUM is provided, we will also
 # create an image with the tag BUILD_NUM.
+# Specify a specific dynamodb version by changing the DYNAMODB_VERSION=latest build flag as desired
```

Comment:
I think a trick we can use here is this:

```make
DYNAMODB_VERSION ?= latest
build:
    docker build -t dwmkerr/dynamodb:latest --build-arg DYNAMODB_VERSION=$(DYNAMODB_VERSION)
    # etc...
```

This way there's not even any need to change the makefile, the caller can just do:

```sh
DYNAMODB_VERSION=1.2.3 make build
```

Or even:

```sh
make build DYNAMODB_VERSION=1.2.3 
```

It's a really nice syntax - if the variable is not specified, the default `latest` is use. This is an excellent approach for CI systems - if you have a specific version in mind you can just set it as an env var and the `makefile` will just pick it up automatically.

(Nice reference here: https://adamcod.es/2016/11/15/makefile-variables.html)